src/python/WMCore/Algorithms/ParseXMLFile.py

```diff
@@ -28,7 +31,7 @@ def __init__(self, name, attrs):
         self.name = str(name)
         self.attrs = {}
         self.text = None
-        [ self.attrs.__setitem__(str(k), str(v)) for k,v in attrs.items()]
+        [ self.attrs.__setitem__(str(k), str(v)) for k,v in viewitems(attrs)]
```

Comment:
```suggestion
        for k, v in viewitems(attrs):
            self.attrs.__setitem__(str(k), str(v))
```
Please check the spaces/indentantion though.

Reply:
Agree with the proposed change here!

Reply:
This is definitely better, Thanks!